app/components/CacheAnalysis.vue

```diff
@@ -32,6 +34,38 @@ const cacheAnalysis = computed(() =>
   getCacheAnalysis(props.cacheHeaders, now.value),
 )
 
+// Helper function to handle hover events on data keys
+const handleDataKeyHover = (dataKey: string, rawValue: unknown) => {
+  // Type guard to ensure we have a valid value for display
+  if (typeof rawValue === 'boolean' || typeof rawValue === 'number' || typeof rawValue === 'string' || rawValue instanceof Date) {
+    const displayValue = getDisplayValue(rawValue)
+    setHover(dataKey, displayValue, rawValue)
```

Comment:
let's only call `setHover`  when there are >1 runs on the page. the point is to compare runs visually but it's just visual noise when there's only one run.